docs/community/training/debugging/techniques-and-tools.md

```diff
@@ -0,0 +1,25 @@
+# Techniques and tools
+
+## Identifying errors
+
+How to read tracebacks?
+
+## Developing a plan
+
+Version control, last known good version, `git bisect`.
+
+## Searching for the root cause
+
+Print statements, debuggers, REPLs, last good version, recent changes, assert statements, comment out code ...
```

Comment:
What do we think of highlighting the "debuggers" in bold capitals to drive home that this is what you should use (the others are okay and help, but im straight to debuggers)

Reply:
This page was a brain dump, and definitely needs to be edited/revised. Yes, I want to drive home the point that debuggers are the most direct means for observing what the code is actually doing, as opposed to what you think it's doing